lib/dor/text_extraction/speech_to_text_cocina_updater.rb

```diff
@@ -6,6 +6,25 @@ module TextExtraction
     class SpeechToTextCocinaUpdater < CocinaUpdater
       private
 
+      # we will skip .json speech to text files, these are only parsed for language to add to cocina
+      def skip_file?(file)
+        file.basename.to_s.start_with?('.') || file.basename.to_s.end_with?('.json')
+      end
+
+      def add_file_to_new_resource(_path)
+        raise 'Dor::TextExtraction::SpeechToTextCocinaUpdater should not be creating new resources for speech to text files'
+      end
+
+      # parse the language from the json file (if found)
+      def language(path)
+        corresponding_json_file = find_workspace_file("#{stem(path)}.json")
+
+        return unless corresponding_json_file
```

Comment:
Shouldn't we always have a JSON file?

Reply:
I would hope so.  Perhaps this could be a raise instead so we know about it.

Reply:
Raised an error instead and added a test.

Reply:
Nice!